### PR TITLE
Light-themed bootstrap modals

### PR DESCRIPTION
**Pull request by bhalt**
This branch is parallel with the `white-modals` branch on ozp-bootstrap-sass. Waiting to push that until approved in HUD/Webtop.

New icon `icon-stack-blueDark` is now in ozp-icons to support this PR.

### CODE REVIEW

File: src/sass/main.scss [hunk 1]
```diff
@@ -319,29 +329,7 @@ body > div.classBanner:last-of-type {
...
-    }
-    > .icon-minimize-10-white{
-      @extend .icon-minimize-10-blue;
-    }
-    > .icon-maximize-10-white{
-      @extend .icon-maximize-10-blue;
-    }
-
-}
+}c
```
Comment (SOAP1914) [inline src/sass/main.scss]: is this the "c" from the deleted word color :)

**Comment by bhalt** [comment]
Good catch @SOAP1914 - removed the stray character with latest commit

**Comment by hoganmaps** [comment]
:+1: looks good in chrome and ie11

**Comment by SOAP1914** [comment]
:+1:

**Pull request merged**